docs/dev/libs/ml/quickstart.md

```diff
@@ -129,6 +129,10 @@ and the [test set here](http://www.csie.ntu.edu.tw/~cjlin/libsvmtools/datasets/b
 This is an astroparticle binary classification dataset, used by Hsu et al. [[3]](#hsu) in their
 practical Support Vector Machine (SVM) guide. It contains 4 numerical features, and the class label.
 
+Before importing the traning and test dataset, Flink SVM only supports threshold binary values of 
+`+1.0` and `-1.0`. Thus a conversion is needed upon downloading the svmguide1 dataset since it is 
+labelled using `1`s and `0`s.
+
```

Comment:
I see. if i understand correctly, summary of the suggestions are:
1. move the explanation of why -1.0 and +1.0 to classification page. 
2. leave the quickstart section clean by adding an explicit conversion from 0,1 to -1,1 so that copy-paste will work. and do not expand too much into why -1,+1 is needed, just mentioned that's needed.

Reply:
By sections I mean `LibSVM files` and `Classification` parts of `quickstart.md`.
I think your explanation of why we need the conversion was good, expanded enough, I just suggested to rephrase its start a bit to be moved to the beginning of `Classification` section. The example of conversion can follow your explanation. The overall structure I suggest:

*LibSVM files*
...Text as before...
```
((( leave only lib SVM importing specifics in this example: )))
val astroTrainLibSVM: DataSet[LabeledVector] = MLUtils.readLibSVM(env, "/path/to/svmguide1")
val astroTestLibSVM: DataSet[LabeledVector] = MLUtils.readLibSVM(env, "/path/to/svmguide1.t")
```
...Text as before..

*Classification*
...Explanation of conversion need before classification...:
```
// conversion code example, e.g. which I suggested
```
...section continues as it was with classification description and its example..

The idea is that at the end user can just copy/paste code snippets starting from the import code, then conversion/normalisation, then classification etc and it eventually works altogether.

Reply:
thx for the detail explanation @azagrebin . Sorry for the previous confusion. I updated the document, please take another look when you have time :-)